### ISSUE TITLE
convert buffer numbers to subscript

### ISSUE THREAD

**Issue by sudomyju** [mengelbrecht/lightline-bufferline#111]
Hi, 
I came up with the idea it would be great to have the ordinal buffer numbers and buffer numbers in superscript and subscript or vice versa delimited with the fraction slash. Still, it seems ls always returns buffer numbers as regular numbers.

Is there some way to do that, please? 

```
let g:lightline#bufferline#show_number = 4   
" number character mappings for buffers      
let g:lightline#bufferline#composed_number_map = {
\ 1:  "\u00b9", 2:  "\u00b2", 3:  "\u00b3", 4:  "\u2074", 5:  "\u2075",
\ 6:  "\u2076", 7: "\u2076", 8: "\u2076", 9: "\u2076", 10: "\u00b9\u2070",
\ 11: "\u00b9\u00b9", 12: "\u00b9\u00b2", 13: "\u00b9\u00b3", 14: "\u00b9\u2074", 15: "\u00b9\u2075",
\ 16: "\u00b9\u2076", 17: "\u00b9\u2077", 18: "\u00b9\u2078", 19: "\u00b9\u2079", 20: "\u00b2\u2070"}
let g:lightline#bufferline#number_map = {    
\ 0: '₀', 1: '₁', 2: '₂', 3: '₃', 4: '₄',    
\ 5: '₅', 6: '₆', 7: '₇', 8: '₈', 9: '₉'}
```
Didn't get me anywhere near. 

Cheers,
su

**Comment by mengelbrecht** [comment]
Hi @sudomyju,

currently only the ordinal buffer numbers are mapped using the `composed_number_map` and `number_map` option. The regular buffer numbers are not mapped.
Your use case is interesting and I would like to support it in some way. However, I am not quite sure yet how this should be configured.

**Comment by sudomyju** [comment]
Hi @mengelbrecht 

Thank you for your answer and interest in the use case. I was taking a look at the _ls_ command and I haven't found any useful information.
I was thinking to write some sort of pairing functionality that will parse ls output and bind it to some character map as you already have it for `composed_number_map` and then supersede the ls output. 

I am not completely fluent in _vimscript_ by now, but it seems to me the `s:filtered_buffers()` call should be altered to look for alternative mapping of the buffer numbers and set up some new records accordingly which will be used for display later.

But that is almost a random guess on my side :-)

Cheers

**Comment by mengelbrecht** [comment]
@sudomyju I implemented this in PR #112. If you switch to the branch `feature/custom-buffer-numbers` you can use superscript numbers for ordinal buffer numbers and subscript for regular buffer numbers using the following settings:
```viml
let g:lightline#bufferline#ordinal_number_map = {
\ 0: '⁰', 1: '¹', 2: '²', 3: '³', 4: '⁴',
\ 5: '⁵', 6: '⁶', 7: '⁷', 8: '⁸', 9: '⁹'}

let g:lightline#bufferline#buffer_number_map = {    
\ 0: '₀', 1: '₁', 2: '₂', 3: '₃', 4: '₄',    
\ 5: '₅', 6: '₆', 7: '₇', 8: '₈', 9: '₉'}
```

Let me know if this solves your use case.

**Comment by mengelbrecht** [comment]
Implemented by #112 which has been merged.
